metadata/src/main/java/org/apache/kafka/controller/ConfigurationControlManager.java

```diff
@@ -423,6 +441,17 @@ public void replay(ConfigRecord record) {
             log.info("Replayed ConfigRecord for {} which set configuration {} to {}",
                     configResource, record.name(), record.value());
         }
+        if (record.name() == TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG) {
```

Comment:
we shouldn't use `==` to compare strings.

Reply:
Sorry. This is bad, really bad :(